graphql/src/schema/api.rs

```diff
@@ -266,10 +265,22 @@ fn field_enum_filter_input_values(
 /// Generates `*_filter` input values for the given list field.
 fn field_list_filter_input_values(
     _schema: &Document,
-    _field: &Field,
-    _field_type: &Type,
-) -> Vec<InputValue> {
-    vec![]
+    field: &Field,
+    field_type: &Type,
+) -> Option<Vec<InputValue>> {
+    let _value_type = ast::get_field_value_type(field_type).ok()?;
```

Comment:
Is the `let _value_type =` assignment necessary at all (it may be)?

Reply:
I'm using `get_field_value_type()` here to check that `field_type` is a ValueType variant (supports filters.)